electrical/breakbeam.py

```diff
@@ -2,27 +2,29 @@
 import time
 from enum import Enum
 
-
 class Breakbeam:
-    ACCEPTABLE_TIME = 3 # time to wait to confirm a full break
-    class BEAM_PINS(Enum):
-        HALF1 = 17
-        HALF2 = 22
-        FULL1 = 23
-        FULL2 = 24
+    ACCEPTABLE_TIME = 3  # time to wait to confirm a full break
+
+    # class BEAM_PINS(Enum):
+    #     HALF1 = 17
+    #     HALF2 = 22
+    #     FULL1 = 23
+    #     FULL2 = 24
 
```

Comment:
Why did we remove this class? It looks like it is needed in sensor modules.

Reply:
This was the hardcoded issue that wouldn't allow me to initialize the sensors elsewhere